packages/core/app/pages/Function.js

```diff
@@ -47,6 +47,11 @@ export const Function = ({ name, exports: functionExports, children }) => {
     return valuesCopy;
   };
 
+  const handleScaleToFit = () => {
+    setScaleToFit(!scaleToFit);
+    iframe.current.contentWindow?.dispatchEvent?.(new CustomEvent('scaleToFit', { detail: { scaleToFit: !scaleToFit } }));
```

Comment:
Is this line necessary? Since the state will be updated, then the preview should automatically also trigger. Or maybe that's not why this is here. Let me know!

Reply:
Agree - it's not necessary when the scale to fit ratio is included in the state. I've removed it.